.github/workflows/CI-cache-cleanup.yaml

```diff
@@ -0,0 +1,32 @@
+name: Cache cleanup
+
+run-name: Cache cleanup
+
+on:
+  workflow_dispatch:
+
+jobs:
+  clear-cache:
+    runs-on: warp-ubuntu-latest-x64-2x
+    name: "Clear cache key: '${{ matrix.cache_key }}'"
+    strategy:
+      matrix:
+        cache_key:
+          - build-evm-node
+          - lint-format
+          - machete
+          - build-runtime-benchmarks
+          - unit-test
```

Comment:
What about other cache_keys. I think there are like 8 or 9? like in this PR: https://github.com/zkVerify/zkVerify/pull/307

Reply:
My bad i misunderstood this one with zkverify :)